client/state/sites/selectors/can-current-user-use-customer-home.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import canCurrentUser from 'state/selectors/can-current-user';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getSite from './get-site';
+
+/**
+ * Returns true if the current user can see the CustomerHome menu item and the corresponding section.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether user can access the Earn section.
```

Comment:
Assuming this is unintentional :) 

```suggestion
 * @return {?Boolean}        Whether user can access the CustomerHome section.
```

Reply:
Right. Although it should be Customer*space*Home. It looks like when I did a massive update I replaced Earn with CustomerHome and it ended up like that. Thanks for catching this.